style.css

```diff
@@ -492,3 +521,232 @@ form textarea {
   color: #5e6c84;
   border: 1px solid #c1c7d0;
 }
+
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+  #menu-icon {
+    display: none;
+  }
+
+  a {
+    text-decoration-line: none;
+    color: #344563;
+  }
+
+  .container {
+    padding-left: 5%;
+    padding-right: 5%;
+  }
+
+  .fixed-top {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 1030;
+  }
+
+  .navbar {
+    padding-top: 0;
+    padding-bottom: 0;
+    width: 100%;
+    margin: 0;
+  }
```

Comment:
- [ ] Kindly make sure the navigation bar is aligned correctly as the one on the Figma design. Please look carefully you will find that the navigation section has the same spacing (padding or margin) on both sides as the same the works section container. It means they are vertically aligned on the same starting point. Kindly make sure to adjust the spacing on both sides of the navigation bar in order to meet the Figma design as much as possible.

   - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/222295775-23f6a6c5-92c5-4ded-a09a-da5f82576c23.png)

   - ### FIGMA DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/222295866-949a7d82-58b6-48e4-ac52-c33be5542405.png)